From 39d3d78d1c66090010d425a8e2b98cc5031df05f Mon Sep 17 00:00:00 2001
From: Yasufumi Ogawa <ogawa.yasufumi@lab.ntt.co.jp>
Date: Mon, 27 May 2019 11:18:51 +0900
Subject: [PATCH 2/9] spp_pcap: rename file of util libs

As renamed in shared dir, this update is to rename `command_dec.*` and
`command_proc.*` to `cmd_parser.*` and `cmd_runner.*`.

Signed-off-by: Yasufumi Ogawa <ogawa.yasufumi@lab.ntt.co.jp>
---
 src/pcap/Makefile                         | 2 +-
 src/pcap/{command_dec.c => cmd_parser.c}  | 2 +-
 src/pcap/{command_dec.h => cmd_parser.h}  | 0
 src/pcap/{command_proc.c => cmd_runner.c} | 4 ++--
 src/pcap/{command_proc.h => cmd_runner.h} | 0
 src/pcap/spp_pcap.c                       | 4 ++--
 6 files changed, 6 insertions(+), 6 deletions(-)
 rename src/pcap/{command_dec.c => cmd_parser.c} (99%)
 rename src/pcap/{command_dec.h => cmd_parser.h} (100%)
 rename src/pcap/{command_proc.c => cmd_runner.c} (99%)
 rename src/pcap/{command_proc.h => cmd_runner.h} (100%)

diff --git a/src/pcap/Makefile b/src/pcap/Makefile
index e5e0b00..61f6f81 100644
--- a/src/pcap/Makefile
+++ b/src/pcap/Makefile
@@ -16,7 +16,7 @@ SPP_WKT_DIR = ../shared/secondary/spp_worker_th
 # all source are stored in SRCS-y
 SRCS-y := spp_pcap.c
 SRCS-y += cmd_utils.c
-SRCS-y += command_proc.c command_dec.c
+SRCS-y += cmd_runner.c cmd_parser.c
 SRCS-y += ../shared/common.c
 SRCS-y += $(SPP_WKT_DIR)/command_conn.c
 SRCS-y += $(SPP_WKT_DIR)/spp_port.c
diff --git a/src/pcap/command_dec.c b/src/pcap/cmd_parser.c
similarity index 99%
rename from src/pcap/command_dec.c
rename to src/pcap/cmd_parser.c
index 943ab0f..6734c1e 100644
--- a/src/pcap/command_dec.c
+++ b/src/pcap/cmd_parser.c
@@ -9,7 +9,7 @@
 #include <rte_log.h>
 #include <rte_branch_prediction.h>
 
-#include "command_dec.h"
+#include "cmd_parser.h"
 
 #define RTE_LOGTYPE_SPP_COMMAND_DEC RTE_LOGTYPE_USER2
 
diff --git a/src/pcap/command_dec.h b/src/pcap/cmd_parser.h
similarity index 100%
rename from src/pcap/command_dec.h
rename to src/pcap/cmd_parser.h
diff --git a/src/pcap/command_proc.c b/src/pcap/cmd_runner.c
similarity index 99%
rename from src/pcap/command_proc.c
rename to src/pcap/cmd_runner.c
index 2987cad..f51b819 100644
--- a/src/pcap/command_proc.c
+++ b/src/pcap/cmd_runner.c
@@ -10,8 +10,8 @@
 #include "shared/secondary/spp_worker_th/string_buffer.h"
 #include "spp_pcap.h"
 #include "shared/secondary/spp_worker_th/command_conn.h"
-#include "command_dec.h"
-#include "command_proc.h"
+#include "cmd_parser.h"
+#include "cmd_runner.h"
 
 #define RTE_LOGTYPE_SPP_COMMAND_PROC RTE_LOGTYPE_USER2
 
diff --git a/src/pcap/command_proc.h b/src/pcap/cmd_runner.h
similarity index 100%
rename from src/pcap/command_proc.h
rename to src/pcap/cmd_runner.h
diff --git a/src/pcap/spp_pcap.c b/src/pcap/spp_pcap.c
index b048237..b48caf4 100644
--- a/src/pcap/spp_pcap.c
+++ b/src/pcap/spp_pcap.c
@@ -15,8 +15,8 @@
 #include "shared/common.h"
 #include "cmd_utils.h"
 #include "spp_pcap.h"
-#include "command_proc.h"
-#include "command_dec.h"
+#include "cmd_runner.h"
+#include "cmd_parser.h"
 #include "shared/secondary/spp_worker_th/spp_port.h"
 
 /* Declare global variables */
-- 
2.17.1

